sycl/source/detail/event_impl.hpp

```diff
@@ -313,7 +315,7 @@ class event_impl {
   std::unique_ptr<HostProfilingInfo> MHostProfilingInfo;
   void *MCommand = nullptr;
   std::weak_ptr<queue_impl> MQueue;
-  const bool MIsProfilingEnabled = false;
+  bool MIsProfilingEnabled = false;
```

Comment:
Is this member ever set to `true`? I'm not getting any search hits for `MIsProfilingEnabled` or `setProfilingEnabled()` where anything other than `false` is used.

Reply:
This member is set in the `event_impl` constructor (https://github.com/intel/llvm/blob/sycl/sycl/source/detail/event_impl.cpp#L159), but it must be reset if the graph profiling property is not set be users.

Reply:
makes sense ta, I misread the diff and thought we were adding this member